source/server/listener_socket_option_impl.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <netinet/tcp.h>
+
+#include "envoy/api/v2/listener/listener.pb.h"
+
+#include "common/network/listen_socket_impl.h"
+#include "common/network/socket_option_impl.h"
+
+#include "server/configuration_impl.h"
+
+#ifdef TCP_FASTOPEN
```

Comment:
I think this can live in `socket_option_impl.h` alongside the others.

Reply:
done